src/components/ResourceCard.js

```diff
@@ -84,6 +88,7 @@ export default function ResourceCard({
     readyToFetch: false,
   })
   const cardResourceId = (resourceId === 'twl') && (viewMode === 'markdown') ? 'tw' : resourceId
+  const IS_RESOURCE_TSV = ['tn', 'tq', 'twl'].includes(cardResourceId)
```

Comment:
Usually we use upper case for constants that never change at runtime, but this is dependent on a variable.  But I suppose the argument could be made that once it is set, it will not change for the card in the current GWE implementation.